worker/lib/workers/qemu.ts

```diff
@@ -70,6 +70,15 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 			throw new Error(`Kernel IP forwarding required for virtualized device networking, enable with 'sysctl -w net.ipv4.ip_forward=1'`);
 		}
 
+		const interfaces = JSON.parse((await execProm(`ip -json a`)).stdout);
+		interfaces.forEach((elem: any) => {
+			elem.addr_info.forEach((addr: any) => {
+				if (addr.local === this.qemuOptions.network.bridgeAddress && elem.ifname !== this.qemuOptions.network.bridgeName) {
+					throw new Error(`Address ${this.qemuOptions.network.bridgeAddress} already in use by interface ${elem.ifname}`);
+				}
+			});
+		});
+
```

Comment:
This is probably not necessary, as on line 262, we handle the case where the assignment fails. For reference, `ip` prints to stderr: `ENOENT 2 No such file or directory` when this happens.

Reply:
Nope, disregard. `ip` prints out an error when attempting to assign the in use address to the _same_ interface, not to another interface.